proto/pdpb.proto

```diff
@@ -668,3 +679,36 @@ message GetDCLocationInfoResponse {
     // which it's set when the number of the max suffix bits changes.
     Timestamp max_ts = 3;
 }
+
+message GetAllKeyspacesRequest {
+    RequestHeader header = 1;
+}
+
+message GetAllKeyspacesResponse {
+    ResponseHeader header = 1;
+    uint64 keyspace_version = 2;
```

Comment:
Is it better to make version keyspace specific?

Reply:
what do you mean?

Reply:
Looks like the version is global. And if that's true whenever it gets updated all key spaces meta data have to be reloaded.

Reply:
keyspace meta should be relative small. So I think it won't be a problem. And the global version helps simplify the synchronization process.